src/pages/home/report/ReportActionCompose/Suggestions.tsx

```diff
@@ -111,7 +111,8 @@ function Suggestions(
 
     const onSelectionChange = useCallback((e: NativeSyntheticEvent<TextInputSelectionChangeEventData>) => {
         const emojiHandler = suggestionEmojiRef.current?.onSelectionChange?.(e);
-        return emojiHandler;
+        const mentionHandler = suggestionMentionRef.current?.onSelectionChange?.(e);
+        return emojiHandler ?? mentionHandler;
```

Comment:
I tried to test a corner case with precondition - have a user with `Eric :smile:` as the first name and empty second name.

By pasting text `@eric :sm` into composer, I got both two suggestion menus at the same time.
<img width="300" alt="image" src="https://github.com/Expensify/App/assets/117511920/67d1abc4-0b2c-4360-adbe-50913bf49283"><img width="300" alt="image" src="https://github.com/Expensify/App/assets/117511920/8f609bf4-ad00-4f81-8e2d-dfee5ca12c53">

As prod has same issue, so I think we can also skip this one in this PR.

Reply:
Yeah, I always wondering whether both suggestions will show or not since we don't return anything inside `onSelectionChange`.